source/framework/vx_node_api.c

```diff
@@ -489,6 +497,14 @@ VX_API_ENTRY vx_node VX_API_CALL vxAccumulateSquareImageNode(vx_graph graph, vx_
                                    dimof(params));
 }
 
+VX_API_ENTRY vx_node VX_API_CALL vxAccumulateSquareImageNodeX(vx_graph graph, vx_image input, vx_uint32 shift, vx_image accum)
```

Comment:
This may work since you modified the vxAccumulateSquareImageNode function, however, the kernel string will not match if queried.  We may need to change the kernel wrapper further to abstract this delta.

Reply:
same as above

Reply:
Same as above

Reply:
After further discussion, it was clarified that the new nodes will use the original kernel names.  Both are the same, only difference is if we use a scalar or integral type to the public API.